src/components/ui/Navbar/Navbar.tsx

```diff
@@ -0,0 +1,302 @@
+        : prev.includes(id)
+          ? prev.filter((drop) => drop !== id)
+          : [...prev, id]
+    );
+
+    if (id !== 'academix') setAcademixDropdowns(false);
+  };
+
+  const toggleMobileMenu = () => setIsMobileMenuOpen((prev) => !prev);
+
+  const handleKeyDown = (
+    event: React.KeyboardEvent<HTMLElement>,
+    action: () => void
+  ) => {
+    if (event.key === 'Enter' || event.key === ' ') {
+      action();
+    }
+  };
+
+  const DropdownItems = (items: { link: string, title: string; }[]) =>
+    items.map(({ title, link }) => (
+      <li
+        key={title}
+        className="cursor-pointer p-2"
+        tabIndex={0}
+        role="menuitem"
+      >
+        {/* for navigation routings */}
+        <Link href={link}>{title}</Link>
+      </li>
+    ));
+
+  return (
+    <nav className="bg-white py-3 px-6 shadow-[0_4px_20px_rgba(0,0,0,0.1)] rounded-[100px] flex justify-between items-center">
+      <div className="text-blue-900 font-bold text-xl flex items-center">
+        <Image src={logo} alt="Logo" width={90} height={90} />
+      </div>
+
+      {/* Toggle icons in mobile view */}
+      <div className="md:hidden">
+        <button
+          onClick={toggleMobileMenu}
+          className="text-blue"
+          aria-label="Toggle mobile menu "
+        >
+          {isMobileMenuOpen ? <IoClose size={40} /> : <HiBars3 size={40} />}
+        </button>
+      </div>
+
+      {/* Desktop menu */}
+      <ul
+        className="hidden md:flex space-x-6 text-blue font-semibold"
+        role="menu"
+      >
+        {/* Home and About Links */}
+        {navItems.slice(0, 2).map(({ title, link }) => (
+          <ul className="cursor-pointer font-semibold" key={title}>
+            <Link href={link}>{title}</Link>
+          </ul>
+        ))}
+
+        {/* Offices Dropdown */}
+        <li className="relative">
+          <button
+            onClick={() => toggleDropdown('offices', true)}
+            className="flex items-center"
+            aria-expanded={openDropdowns.includes('offices')}
+            aria-controls="offices-dropdown"
+          >
+            <span>Offices</span>
+            <div className="mt-1">
+              {openDropdowns.includes('offices') ? (
+                <IoMdArrowDropup size={ArrowSize} />
+              ) : (
+                <IoMdArrowDropdown size={ArrowSize} />
+              )}
+            </div>
+          </button>
+          {openDropdowns.includes('offices') && (
+            <div
+              id="offices-dropdown"
+              className="absolute space-y-[-4px] font-normal left-1/2 transform -translate-x-1/2 bg-white w-[255px] text-center shadow-[0_4px_20px_rgba(0,0,0,0.1)] rounded-[20px] py-3 px-7 mt-10 z-10"
+              role="menu"
+            >
+              {DropdownItems(officesDropdownItems)}
+            </div>
+          )}
+        </li>
+
+        {/* Central Board and FAQs Links */}
+        {navItems.slice(2).map(({ title, link }) => (
```

Comment:
Separate the nav items into two separate constants instead of slicing. This will break the code if you add additional elements in the constant.